file-formats/smbc/type/zif_aff_smbc_v1.intf.abap

```diff
@@ -0,0 +1,210 @@
+      "! Toggle between onepage concept and tabs.
+      "! $values {@link zif_aff_smbc_v1.data:co_section_layout}
+      "! $default {@link zif_aff_smbc_v1.data:co_section_layout.page}
+      section_layout     TYPE string,
+      "! <p class="shorttext synchronized">Table Creation Mode</p>
+      "! $values {@link zif_aff_smbc_v1.data:co_creation_mode_name}
+      "! $default {@link zif_aff_smbc_v1.data:co_creation_mode_name.new_page}
+      creation_mode_name TYPE ty_creation_mode-name,
+      "! <p class="shorttext synchronized">Create at End</p>
+      "! If true, a new row is added to the end of the table instead of the beginning of the table
+      create_at_end      TYPE ty_creation_mode-create_at_end,
+      "! <p class="shorttext synchronized">Hide Filter</p>
+      "! If true, the user cannot filter data of the table
+      hide_filter        TYPE ty_personalization-hide_filter,
+      "! <p class="shorttext synchronized">Hide Sort</p>
+      "! If true, the user cannot sort the table
+      hide_sort          TYPE ty_personalization-hide_sort,
+      "! <p class="shorttext synchronized">Hide Column</p>
+      "! If true, the user cannot add and remove columns to the table
+      hide_column        TYPE ty_personalization-hide_column,
+    END OF ty_object_page,
+    "! <p class="shorttext synchronized" >Object Page Configurations</p>
+    ty_object_pages TYPE SORTED TABLE OF ty_object_page WITH UNIQUE KEY entity,
+    "! <p class="shorttext synchronized" >Confguration of List Report and Object Pages</p>
+    "! The Object Page of the Root Entity cannot be configured.
+    "! It is sufficient to maintain only those attributes that should deviate from the standard behavior.
+    BEGIN OF ty_settings,
+      "! $showAlways
+      list_report  TYPE ty_list_report,
+      "! $showAlways
+      object_pages TYPE ty_object_pages,
+    END OF ty_settings.
+  TYPES:
+*        See also {@link if_mbc_cp_api_business_config}
+    "! <p class="shorttext">Service Configuration</p>
+    BEGIN OF ty_service_configuration,
+      "! <p class="shorttext">Service Binding</p>
+      "! Service Binding used for business configuration maintenance
+      "! $required
+      service_binding  TYPE c LENGTH 40,
+      "! <p class="shorttext">Service Name</p>
+      "! The name of the service to be used (as defined in the Service Binding)
+      "! $required
+      service_name     TYPE c LENGTH 40,
+      "! <p class="shorttext">Service Version</p>
+      "! The version of the service to be used (as defined in the Service Binding)
+      "! $required
+      service_version  TYPE n LENGTH 4,
+      "! <p class="shorttext">Root Entity Set</p>
+      "! The name of an entity set as exposed by the Service Definition.
+      "! This entity set is used as the root node for the UI.
+      "! Only for this root entity set and its associations a UI is shown.
+      "! $required
+      root_entity      TYPE c LENGTH 30,
```

Comment:
I am not sure is the addition would be `_set` important? See the field's title

Reply:
_set was added and the descriptions was changed